src/main/java/org/ld4l/bib2lod/managers/SimpleManager.java

```diff
@@ -40,21 +44,32 @@ public static void main(String[] args) {
         LOGGER.info("START CONVERSION.");
 
         try {
-            setup(args);
-            convert();
+        	SimpleManager fm = new SimpleManager(args);
+        	fm.convert();
             LOGGER.info("END CONVERSION.");
         } catch (Exception e) {
             LOGGER.error(e.getMessage());
             e.printStackTrace();
             LOGGER.error("CONVERSION FAILED TO COMPLETE");
         }
     }
-
+    
```

Comment:
SimpleManager, lines 57-63. A private niladic constructor with a do-not-use warning is redundant and confusing. The presence of other constructors mean that the default constructor will not be generated. Or am I missing something? (see "confusing", above)